### ISSUE TITLE
Precompiling ACE1pack

### ISSUE THREAD

**Issue by wcwitt** [ACEsuit/ACEpotentials.jl#131]
On the topic of precompiling, I've grown really tired of seeing this over and over
```
julia> using ACE1pack
[ Info: Precompiling ACE1pack [8c4e8d19-0bd6-4234-8309-7210652e3178]
[ Info: Skipping precompilation since __precompile__(false). Importing ACE1pack [8c4e8d19-0bd6-4234-8309-7210652e3178].
[ Info: Precompiling ASE [51974c44-a7ed-5088-b8be-3e78c8ba416c]
[ Info: Skipping precompilation since __precompile__(false). Importing ASE [51974c44-a7ed-5088-b8be-3e78c8ba416c].
```

Could someone tell me how to fix it, or at least explain what's happening?

**Comment by cortner** [comment]
I already asked Teemu to remove ASE - I don't see that we still need it?

**Comment by wcwitt** [comment]
It appears this is addressed by #134.
